pt/lessons/specifics/nerves.md

```diff
@@ -0,0 +1,262 @@
+
+_Dica: `ssh 192.168.88.2` te dá um shell IEx no dispositivo no contexto da aplicação._
+
+_Solução de Problemas: Se você não tiver uma chave ssh existente em sua pasta home, você vai receber um erro `No SSH public keys found in ~/.ssh.`. Nesse caso, você vai precisar rodar `ssh-keygen` e re-gravar o firmware para usar o recurso de atualização por rede._
+
+## Configurando o controle do LED
+
+Para interagir com LEDs, você vai precisar do pacote [nerves_leds](https://github.com/nerves-project/nerves_leds) instalado, o que é feio adicionando `{:nerves_leds, "~> 0.8", targets: @all_targets},` no arquivo `mix.exs`.
+
+Depois de instalar a dependência, você precisa configurar a lista de LED para o dispositivo. Por exemplo, para todos modelos de Raspberry Pi, existe apenas um LED onboard: `led0`. Vamos usá-lo adicionando uma linha `config :nerves_leds, names: [green: "led0"]` ao arquivo `config/config.exs`.
+
+Para outros dispositivos, você pode dar uma olhada na [parte correspondente do projeto nerves_example](https://github.com/nerves-project/nerves_examples/tree/master/hello_leds/config).
+
+Depois de configurar o LED em si, nós certamente precisamos controlá-lo de alguma forma. Para fazer isso, nós adicionaremos um GenServer (veja detalhes sobre GenServers na lição [Concorrência OTP](../../advanced/otp-concurrency.md)) em `lib/network_led/blinker.ex` com esses conteúdos:
+
+```elixir
+defmodule NetworkLed.Blinker do
+  use GenServer
+
+  @moduledoc """
+    Simple GenServer to control GPIO #18.
+  """
+
+  require Logger
+  alias Nerves.Leds
+
+  def start_link(state \\ []) do
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
+  end
+
+  def init(state) do
+    enable()
+
+    {:ok, state}
+  end
+
+  def handle_cast(:enable, state) do
+    Logger.info("Enabling LED")
+    Leds.set(green: true)
+
+    {:noreply, state}
+  end
+
+  def handle_cast(:disable, state) do
+    Logger.info("Disabling LED")
+    Leds.set(green: false)
+
+    {:noreply, state}
+  end
+
+  def enable() do
+    GenServer.cast(__MODULE__, :enable)
+  end
+
+  def disable() do
+    GenServer.cast(__MODULE__, :disable)
+  end
+end
+
+```
+
+Para habilitar isso, você também precisa adicioná-lo à árvore de supervisão em `lib/network_led/application.ex`: adicione `{NetworkLed.Blinker, name: NetworkLed.Blinker}` sob o grupo `def children(_target) do`.
+
+Note que o Nerves tem duas diferentes árvores de supervisão na aplicação - uma para a máquina hospedeira e uma para os dispositivos de fato.
+
+Depois disso - é isso! Você pode fazer upload do firmware e, ao rodar o IEx através de ssh no dispositivo alvo, checar que `NetworkLed.Blinker.disable()` desliga o LED (que é habilitado por padrão no código), e `NetworkLed.Blinker.enable()` liga.
```

Comment:
so...

_Depois disso - é isso!_

it's confusing to me, maybe change for:

_Depois disso, é só você realizar o upload do firmware..._

what do you think?